### ISSUE TITLE
Activation function for discriminator of WGAN-GP

### ISSUE THREAD

**Issue by yutaochen1998** [timsainb/tensorflow2-generative-models#16]
In the network architecture of discriminator:
`tf.keras.layers.Dense(units=1, activation="sigmoid"),`
You don't need any activation function for discriminator, since you are using Wasserstein Loss. Using sigmoid here would greatly limit your convergeance speed.
